csharp/src/SeedLang/Interpreter/FunctionStack.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+// Copyright 2021-2022 The SeedV Lab.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace SeedLang.Interpreter {
+  // A stack used in the compiler to store compiled functions.
+  internal class FunctionStack {
```

Comment:
This class and its description are a bit hard to understand. And the class name is a bit confusing with `CallStack`. What's a `FunctionStack` required by the compiler? Is the class used for nested functions? If so, I guess it'd be better to name it `NestedFunctionStack`.

Reply:
Yes, this class is used to hold nested functions, and it's a must, because a main function will be generated for the global code. Then any function definition will be a nested function.
I will rename it `NestedFuncStack`.